diff --git a/node_modules/workerd/worker.mjs b/node_modules/workerd/worker.mjs
index a9697a4..72211ce 100755
--- a/node_modules/workerd/worker.mjs
+++ b/node_modules/workerd/worker.mjs
@@ -7488,7 +7488,7 @@ interface RequestInitCfPropertiesImage extends BasicImageTransformations {
    *   in combination with the properties above
    * - border - this will automatically trim the surroundings of an image based on
    *   it's color. It consists of three properties:
-   *    - color: rgb or hex representation of the color you wish to trim (todo: verify the rgba bit)
+   *    - color: rgb or hex representation of the color you wish to trim or rgba CSS4 modern syntax
    *    - tolerance: difference from color to treat as color
    *    - keep: the number of pixels of border to keep
    */
